blockchain/superchain-starter-superchainerc20/contracts/src/TokenMinter.sol

```diff
@@ -1,16 +1,22 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.20;
 
-import "@openzeppelin/contracts/access/AccessControl.sol";
+// import "@openzeppelin/contracts/access/AccessControl.sol";
 import "@openzeppelin/contracts/token/ERC1155/utils/ERC1155Holder.sol";
-import "./NFTMinter.sol";
-import {InitialSupplySuperchainERC20} from "./InitialSupplySuperchainERC20.sol";
+import {IERC1155 as INameWrapper} from "@openzeppelin/contracts/token/ERC1155/IERC1155.sol";
```

Comment:
make custom interface with `function owner(bytes32) returns (address)` for contract `0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e` (same on mainnet and sepolia)

use this to check name ownership after onchain import

Reply:
redirect to https://sepolia.app.ens.domains/ for user to generate registry entry for prev step

Reply:
include .eth names, tokenized ownership of web2 and web3 domain names powered by ens